ui/src/widgets/ui-dropdown/UIDropdown.vue

```diff
@@ -58,6 +62,12 @@ export default {
             set (value) {
                 this.items = value
             }
+        },
+        multiple: function () {
+            return this.dynamic.multiple === null ? this.props.multiple : this.dynamic.multiple
+        },
+        label: function () {
+            return this.dynamic.label || this.props.label
```

Comment:
This will not permit a user to set an empty string label.  e.g. a label of `""` is _falsey_ and will default to `props.label`

Reply:
Addressed in fb646d4e